src/Map/index.tsx

```diff
@@ -81,6 +82,34 @@ const applyLayerDefaults = (props: Props) => {
   }
 }
 
+const updateMap = (sourceJson: GeoJSON, map: mapboxgl.Map) => {
+  if ((sourceJson as FeatureCollection).features.length > 0) {
+    let bounds = (sourceJson as FeatureCollection).features.reduce(function (
+      bounds: any,
+      feature: any
+    ) {
+      if (!Array.isArray(feature.geometry.coordinates[0])) {
+        return bounds.extend(feature.geometry.coordinates)
+      } else {
+        return feature.geometry.coordinates.reduce(function (
+          bounds: any,
+          coord: any
+        ) {
+          return bounds.extend(coord)
+        },
+        bounds)
+      }
+    },
+    new mapboxgl.LngLatBounds())
+    map.fitBounds(bounds, {
+      linear: false,
+      maxZoom: 12,
```

Comment:
could use `onClickZoomIn`? Maybe we change `onClickZoomIn` to `focusZoom` or something as it's used for more than the level to zoom to when clicking, but generally a zoom level when focusing the map